MERN Scripts/mern-emsystem/src/.env

```diff
@@ -0,0 +1,3 @@
+SQUARE_ACCESS_TOKEN=EAAAELql3EigTRh3QbNoSnOuFLzYvuJNZw8AHqmUmk7v4M5Y3Gx9e0JDFUo22mQX
+SQUARE_APPLICATION_ID=sandbox-sq0idb-7JEDuestTK6Hj-fdj6C0FA
+SQUARE_LOCATION_ID=LS2QD8Y1ET9FY
```

Comment:
I'm assuming that these will work on any machine that runs them? i.e. they are not unique to a particular ip or user